src/labels/CustomLabels.labels

```diff
@@ -6823,6 +7208,38 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist User Onboard in Get Started</shortDescription>
         <value>Onboard Your Users</value>
     </labels>
+    <labels>
+        <fullName>gsChecklistItemFundraisingProcessDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item FundraisingProcess in Get Started</shortDescription>
+        <value>Learn how to leverage built-in tools like Path and Kanban to effectively manage your fundraising and donation process and ensure everyone at your organization has a window into how those processes work.</value>
```

Comment:
Please add a comma after donation process. This should read: Learn how to leverage built-in tools like Path and Kanban to effectively manage your fundraising and donation process, and ensure everyone at your organization has a window into how those processes work.